server/v2/appmanager/config.go

```diff
@@ -0,0 +1,7 @@
+package appmanager
+
+type Config struct {
```

Comment:
are these supposed to be passed externally from something outside `appmanager`? If so, all the fields are private.